jira/client.py

```diff
@@ -716,6 +730,8 @@ def _fetch_pages(
             page_params["startAt"] = startAt
         if maxResults:
             page_params["maxResults"] = maxResults
+        elif batch_size := self._get_batch_size(item_type):
```

Comment:
is this the first walrus operator we're using?
somebody has to go first :)
I must admit I'm not used to it yet

Reply:
It's an honor (for some). Just have to be very careful about braces when using it..